src/app/lib/utilities/blockHandlers/index.js

```diff
@@ -2,8 +2,15 @@
 // Contains various helper functions for managing props as well as finding specific
 // elements and blocks within them
 
-// Filters array of blocks for a single block of given type
-const filterForBlockType = (arrayOfBlocks, type) =>
-  arrayOfBlocks?.filter(block => (block ? block.type === type : null))?.[0];
+// filters an array of blocks for a given type and Returns the first match by default
+// pass `{ multiple: true }` to get all matching blocks.
+const filterForBlockType = (
+  arrayOfBlocks,
+  type,
+  options = { multiple: false },
+) => {
+  const filtered = arrayOfBlocks?.filter(block => block?.type === type);
+  return options.multiple ? filtered : filtered?.[0];
+};
```

Comment:
Felt funny about updating this helper but by passing in the `options` we were hoping to add a non-breaking change to support existing media but also allow this single helper to support the block structure of `portrait` videos in the carousel. We set these options to true for the portrait video config but might be able to refactor the config somehow

Reply:
@amoore108 take it this was fine?

Reply:
we have created a new `index.test.js` co-located with 

https://github.com/bbc/simorgh/blob/838e013393ed912a363ffd468cb0236a8a33b31e/src/app/lib/utilities/blockHandlers/index.js

As it's not obvious where the best place to test would be and there's not currently a test file in the utility subfolder. 

Got 5 test cases in there now - hopefully they're all good?